### DISCUSSION TITLE
What am I doing wrong here?

### DISCUSSION THREAD

**Discussion by pjb3** [js-fns/date-fns#1966] [General]
Say I have a string with a US/Eastern offset in it. I can parse it into a utcDate like this:

```
> const { format, zonedTimeToUtc } = require("date-fns-tz")
undefined
> const utcDate = zonedTimeToUtc('2020-09-21T11:50:00-04:00')
undefined
> utcDate
2020-09-21T15:50:00.000Z
```

Now if I try to use format to print it back out in US Eastern, I get this:

```
> format(utcDate, 'yyyy-MM-dd HH:mm z OOOO', { timeZone: 'US/Eastern' })
'2020-09-21 08:50 EDT GMT-04:00'
```

Why isn't that `2020-09-21 11:50 EDT`? This is in a node.js console and my time zone is US/Pacific.

**Comment by michaelyuen** [comment]
I think your format string is incorrect. `O` (letter) is for `Timezone (GMT)`: https://date-fns.org/v2.16.1/docs/format

Just removing that portion, the `z` should do what you want: `'yyyy-MM-dd HH:mm z'`

I'd also recommend using `America/New_York` as `US/Eastern` is deprecated: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Reproduction: https://codesandbox.io/s/date-fns-tz-question-4n9u4
